LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Trakx
+Copyright (c) 2024 Trakx
```

Comment:
Btw, I think we are removing the LICENSE file, since we don't have a license at all.
I think it's something we can discuss in our meeting. 😉

Reply:
This is a public repo, so I think we should keep a license, no ?

Reply:
Hmm, I didn’t notice it is public. Maybe we should check if we use MIT based license, Apache or an own license I think. Definitely something to discuss with the team.